invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/__tests__/document.js

```diff
@@ -17,22 +17,22 @@ describe('Document query builder tests', () => {
     expect(query).toEqual('circulation.pending_loans:>0');
   });
 
-  it('should build query with loanable items', () => {
+  it('should build query with available items', () => {
```

Comment:
`available`? Isn't it something we wanted to get rid?

Reply:
we wanted to get rid of loanable! any better ideas how to call those items?

Reply:
I agree, `available` is a bit misleading. Some suggestions: `circulatable` or `circulable`